examples/multi_backend/src/multi_backend.cr

```diff
@@ -0,0 +1,48 @@
+require "turbo/cable"
```

Comment:
Since `Cable` adopted a bunch of the customizations I'd made to my fork, I didn't have to make a single change to my `turbo/cable` integration! 💯